src/celeritas/ext/VecgeomParams.cc

```diff
@@ -120,26 +155,27 @@ VecgeomParams::VecgeomParams(const std::string& filename)
     {
         auto& cuda_manager = vecgeom::cxx::CudaManager::Instance();
 
-        CELER_LOG(debug) << "Converting to CUDA geometry";
+        CELER_LOG(info) << "Converting to CUDA geometry";
         {
             ScopedTimeAndRedirect time_and_output_("vecgeom::CudaManager");
             // cuda_manager.set_verbose(1);
             cuda_manager.LoadGeometry();
             CELER_DEVICE_CALL_PREFIX(DeviceSynchronize());
         }
 
-        CELER_LOG(debug) << "Transferring geometry to GPU";
+        CELER_LOG(info) << "Transferring geometry to GPU";
         {
             ScopedTimeAndRedirect time_and_output_("vecgeom::CudaManager");
             auto world_top_devptr = cuda_manager.Synchronize();
             CELER_ASSERT(world_top_devptr != nullptr);
             device_ref_.world_volume = world_top_devptr.GetPtr();
             device_ref_.max_depth    = host_ref_.max_depth;
+            CELER_DEVICE_CALL_PREFIX(DeviceSynchronize());
             CELER_DEVICE_CHECK_ERROR();
         }
         CELER_ENSURE(device_ref_);
 
-        CELER_LOG(debug) << "Initializing BVH on GPU";
+        CELER_LOG(info) << "Initializing BVH on GPU";
```

Comment:
Revert to debug, and if you want to temporarily add more debugging info for your tests, just `export CELER_LOG=debug`